src/attributes.jl

```diff
@@ -64,8 +63,10 @@ read_attribute(attr::Attribute, memtype::Datatype, buf) = API.h5a_read(attr, mem
 
 Open the [`Attribute`](@ref) named `name` on the object `parent`.
 """
-open_attribute(parent::Union{File,Object}, name::AbstractString, aapl::AttributeAccessProperties=AttributeAccessProperties()) =
-    Attribute(API.h5a_open(checkvalid(parent), name, aapl), file(parent))
+open_attribute(parent::Union{File,Object}, name::AbstractString, aapl::AttributeAccessProperties=AttributeAccessProperties()) = Attribute(API.h5a_open(checkvalid(parent),
```

Comment:
I'm wondering if this should be a one liner or not, since it formats really strangely.

Reply:
Those are a `yas` effect. Those look better in `blue`.